github/github.go

```diff
@@ -82,8 +83,10 @@ func (c *Config) NewClient() (*Client, error) {
 		transport = tr
 	}
 
-	transport = metrics.Transport{Transport: transport}
-	httpClient := &http.Client{Transport: transport}
+	cached := httpcache.NewTransport(httpcache.NewMemoryCache())
+	cached.Transport = transport
+	metricsTransport := metrics.Transport{Transport: cached}
```

Comment:
Perhaps this ordering results in the metrics not being distinguished between cached vs not-cached. But that's another story.